src/CodeGen_PTX_Dev.cpp

```diff
@@ -430,9 +430,15 @@ vector<char> CodeGen_PTX_Dev::compile_to_src() {
     // Output string stream
 
     // Ask the target to add backend passes as necessary.
+#if LLVM_VERSION < 70
```

Comment:
super-nit: don't we usually prefer to do LLVM_VERSION >= instead?

Reply:
I see a mix of both, and more usages of `LLVM_VERSION < x` than `LLVM_VERSION >= x`: https://github.com/halide/Halide/search?p=1&q=LLVM_VERSION&unscoped_q=LLVM_VERSION